src/current/v24.2/stream-a-changefeed-to-amazon-msk-serverless.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Stream a Changefeed to Amazon MSK Serverless
+summary: Learn how to connect a changefeed to stream data to an Amazon MSK Serverless cluster.
+toc: true
+---
+
+{% include_cached new-in.html version="v24.2" %} [Changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}) can stream change data to [Amazon MSK Serverless clusters](https://docs.aws.amazon.com/msk/latest/developerguide/serverless.html) (Amazon Managed Streaming for Apache Kafka), which is an Amazon MSK cluster type that automatically scales your capacity.
+
+MSK Serverless requires [IAM authentication](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html?icmpid=docs_iam_console) for the changefeed to connect to the cluster.
+
+In this tutorial, you'll set up an MSK Serverless cluster and connect a changefeed with IAM authentication. You'll create the MSK Serverless cluster with an IAM policy and role. CockroachDB and a Kafka client will assume the IAM role in order to connect to the MSK Serverless cluster. Then you'll set up the Kafka client to consume the changefeed messages and start the changefeed on the CockroachDB cluster.
+
+An overview of the workflow involves:
```

Comment:
i feel like this list kind of says the previous paragraph again and also duplicates the TOC

could prolly be deleted IMO but that's just, like, an opinion